PC/pyconfig.h

```diff
@@ -498,7 +502,9 @@ Py_NO_ENABLE_SHARED to find out.  Also support MS_NO_COREDLL for b/w compat */
 /* #define HAVE_CLOCK */
 
 /* Define when any dynamic module loading is enabled */
-#define HAVE_DYNAMIC_LOADING
+#ifndef MS_WINDOWS_GAMES
+#  define HAVE_DYNAMIC_LOADING
```

Comment:
I think I did this in my original patch simply because I did not need it. I would assume dynamic loading to be generally possible. I will give it a try.

Reply:
Compiles without any issues, so I guess this was just copied from the playstation build :)